src/lib/components/Flowbite/Flowbite.tsx

```diff
@@ -33,6 +32,14 @@ export const Flowbite: FC<FlowbiteProps> = ({ children, theme = {} }) => {
       if (windowExists()) {
         document.documentElement.classList.add('dark');
       }
+    } else if (dark === false) {
```

Comment:
Can we simplify this whole branching section inside the `useEffect`? I think the reason we were specifically targeting `false` here was because this was being stored in local storage before which can technically be empty.